include/IECore/MurmurHash.inl

```diff
@@ -334,6 +342,31 @@ inline void murmurHashAppend( IECore::MurmurHash &h, const std::vector<T> &data
 	h.append( data.data(), data.size() );
 }
 
+// vector<bool> is a special case because we can't get a pointer to the internal data,
+// because it's stored packed
+template<>
+inline void murmurHashAppend( IECore::MurmurHash &h, const std::vector<bool> &data )
+{
+	// Repack the data into something we can access and hash.
+	// We use a 16 char array on the stack because MurmurHash operates on chunks
+	// of 128 bits, so this should get good throughput without needing to alloc
+	// storage for the whole vector
+	unsigned char p[16];
+	for( size_t i=0; i<data.size(); i+=128 )
+	{
+		memset( p, 0, 16 );
+		for( int j = 0; j < std::min( 128, (int)(data.size() - i) ); j++ )
+		{
+			p[j/8] |= data[i + j] << j;
```

Comment:
Eww, that's embarrassing.  At least I was feeling uneasy enough about this code that I wanted to add a test, but after catching the previous issue with not including size(), I didn't catch that including size then invalidated much of the rest of the test.  I've squashed in the fix, since it was just replacing `<< j` with `<< (j%8)`.